Integrations/integration-SplunkPy.yml

```diff
@@ -57,6 +57,60 @@ script:
     import urllib2
     import ssl
     from StringIO import StringIO
+    import requests
+    import urllib3
+    urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+    def updateNotableEvents(sessionKey, baseurl, comment, status=None, urgency=None, owner=None, eventIDs=None, searchID=None):
+        """
+        Update some notable events.
+
+        Arguments:
+        sessionKey -- The session key to use
+        comment -- A description of the change or some information about the notable events
+        status -- A status (only required if you are changing the status of the event)
+        urgency -- An urgency (only required if you are changing the urgency of the event)
+        owner -- A nowner (only required if reassigning the event)
+        eventIDs -- A list of notable event IDs (must be provided if a search ID is not provided)
+        searchID -- An ID of a search. All of the events associated with this search will be modified unless a list of eventIDs are provided that limit the scope to a sub-set of the results.
```

Comment:
I have not seen any place that we pass `searchID`

I guess that this code was taken from somewhere?

Reply:
yep, will support in next version of this.. for now, not sure it will be needed